Resources/config/doctrine_mongodb_admin.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
+        <!-- COMMON -->
+        <parameter key="sonata.media.admin.groupname">sonata_media</parameter>
```

Comment:
I don't know what interest it has exactly, but if you find one, then make sure it appears in your commit message. As a rule of thumb, try to have commit messages that look like this : 

```
What you are doing
<blank line>
Why you are doing it
```
More information about commit messages can be found in the CONTRIBUTING.md

Anyway, nice first PR, it is quite clean (cleaner than the average PR IMO).